[Do not merge] Document ignore exit-code functionality (#38984)
dotnet/docs: docs/core/testing/unit-testing-mstest-runner-intro.md

Prints out a description of how to use the command.

- **`-ignore-exit-code`**

> Allows to consider some non-zero exit codes as `0`. For more information, refer to [Ignoring specific exit codes](./unit-testing-mstest-runner-exit-codes.md#ignoring-specific-exit-codes).

Reviewer: ```suggestion
Allows some non-zero exit codes to be ignored, and instead returned as `0`. For more information, see [Ignore specific exit codes](./unit-testing-mstest-runner-exit-codes.md#ignore-specific-exit-codes).
```
